### DISCUSSION TITLE
Microsoft Hackathon 2024

### DISCUSSION THREAD

**Discussion by jonathanpeppers** [jonathanpeppers/dotnes#49] [Announcements]
I'm working on this full time from Sept 16, 2024 through Sept 20, 2024!

If you want to help, reach out!

**Comment by magmukendi** [comment]
which hackathon is this ? can we have more info ?

**Reply by jonathanpeppers** [reply]
It’s a hackathon for Microsoft Employees, but this is just an fyi I have more time to help this week if anyone is interested in contributing. Thanks!

**Reply by magmukendi** [reply]
I see. I thought this was some sort of "open to all" type of hackathon :)

**Comment by jonathanpeppers** [comment]
Just kind of wrapping up for the week, but if anyone would like to join our new discord, it's at: https://discord.gg/YbAyj6KP
